designs/service_function_chaining/SFC_design.md

```diff
@@ -46,7 +46,29 @@ SFC functionality in OSC will adapt to existing SDN controller SDK APIs.SFC conf
 ![](./images/sw-modules.png)
 ### REST API   
 Currently not applicable.
- 
+
+### OSC Configuration Changes
+OSC security group binding configuration for SFC will be different compare to current configuration steps. For binding a security group with  Service Function in Openstack using SFC, user has to create chain by selecting the DAs in a VC.
+Each DA in a VC is a service function. And user should be able to select DAs in particular order to create a service function chaining.  
+
+The step are show below. And steps in bold represent changes in OSC for SFC functionality.  
+
+1. DA Creation (A service function chain is one or more DAs)
+1. **Port Chain Creation ( Create service function chaining)**
+1. Create a security group
+1. **Binding a Security Group**
+
+#### Port Chain
+A new configuration requirement, Port Chain is introduced for SFC functionality. In this configuration an OSC entity called PORT_CHAIN will be created and this will have a reference point to corresponding Virtual connector DAs. In OSC DB a PORT_CHAIN table and corresponding related fields of entities will be create. And this Database table will have reference to DA's attached to this chain.At this point of time Port Chain is setup only in OSC and not action is taken place at target Openstack VC.  
+
+**OSC Entities Mapping**
+![](./images/pc-mapping.png)

```

Comment:
In this section was only mentioning how OSC modules are related to one another for SFC configuration. like how VC, DA and SG config entities are connected to port chain. Do not want to relate to any database. I did rephrased the lines a bit..